File: respy/fortran/logging/logging_warning.f90
!******************************************************************************
!******************************************************************************
MODULE logging_warning

	!/*	external modules	*/

    USE shared_containers 

    USE shared_constants

    USE shared_auxiliary 

    USE shared_utilities

	!/*	setup	*/

    IMPLICIT NONE

    PUBLIC

CONTAINS
!******************************************************************************
!******************************************************************************
SUBROUTINE log_warning_crit_val()

!------------------------------------------------------------------------------
! Algorithm
!------------------------------------------------------------------------------
    
    OPEN(UNIT=99, FILE='est.respy.log', ACCESS='APPEND')

        WRITE(99, *) '  Warning: Value of criterion function too large to write to file, internals unaffected.'

    CLOSE(1)


END SUBROUTINE
!******************************************************************************
!******************************************************************************
END MODULE

File: respy/python/logging/logging_warning.py


def log_warning_crit_val(count):

    with open('est.respy.log', 'a') as out_file:

        line = '   Warning: '

        if count == 0:
            line += 'Starting '
        if count == 1:
            line += 'Step '
        if count == 2:
            line += 'Current '

        line += 'value of criterion function too large to write to file, ' \
                'internals unaffected.\n'

        out_file.write(line)
